### DISCUSSION TITLE
Components inheriting from others good practice?

### DISCUSSION THREAD

**Discussion by MrCheeseWasTaken** [skypjack/entt#860] [Q&A] [answered]
I am using sfml and want to make a transform component with enTT using sf::Transformable
If I do `struct TransformComponent : public sf::Transformable` is this practice to use with enTT?
or should the TransformComponent not inherit anything and just have sf::Transformable as a member variable

**Comment by skypjack** [comment, marked as answer]
The only requirement for your components is that they are movable. What remains out is a matter of tastes.
